### PR TITLE
Add details on how to set agent & collector endpoints

### LINKED ISSUE #122: Document how to specify agent/collector endpoint
Hey guys

I'm trying to add jaegertracing to a C++ microservice running on an Istio mesh.

I got working the example app but I can't find a way to specify what is the collector endpoint the app will use to send the spans/traces and the only loggers I see defined do nothing or just print on screen the messages.

Can someone tell me what would be the config parameter that defines the collector to use? 
Or point me to some working example/documentation to use as a guide?

Thanks for you help

### PR DESCRIPTION
**Pull request by MikeGoldsmith**
## Which problem is this PR solving?
Resolves #122

## Short description of the changes
Adds details on how to set the agent host:port and the collector URL in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,26 @@ The code can be re-generated with
...
 
+### Updating Agent host and port
+
+The default agent host and port is `127.0.0.1:6831`, you can use a different host/port by updating your reporter configuration.
+
+YAML configuration:
+
+```yml
+repoter:
+  localAgentHostPort: my-jaeger-agent.local # or another IP:Port 10.120.1.101:6831
```
Comment (yurishkuro) [inline README.md]: `my-jaeger-agent.local` looks very confusing to me, what does it mean? Shouldn't there be a port number?

File: README.md [hunk 2]
```diff
@@ -53,6 +53,26 @@ The code can be re-generated with
...
+```
+
+### Collector URL
```
Comment (yurishkuro) [inline README.md]: perhaps "Sampling Config Server URL"?

File: README.md [hunk 3]
```diff
@@ -53,6 +53,26 @@ The code can be re-generated with
     $ git apply scripts/thrift-gen.patch
 ```
 
+### Updating Agent host and port
+
+The default agent host and port is `127.0.0.1:6831`, you can use a different host/port by updating your reporter configuration.
```
Comment (yurishkuro) [inline README.md]: we should mention that UDP transport should only be used on the same host

**Comment by MikeGoldsmith** [comment]
@yurishkuro I've applied your suggestions

**Pull request merged**